vermouth/molecule.py

```diff
@@ -327,6 +336,15 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.interactions = defaultdict(list)
 
+    def __eq__(self, other):
+        return (
+            self.nrexcl == other.nrexcl
+            and self._force_field == other._force_field
+            and self.same_nodes(other)
+            and set(self.edges) == set(other.edges)
```

Comment:
Not good enough. Edges are undirected, so `(1, 2) != (2, 1)`

Reply:
Beyond this, it does not account for edge attributes yet.